app/src/login.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./login.css";
+
+async function loginUser(credentials) {
+  return fetch("http://localhost:1000/login", {
+    method: "POST",
+
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(credentials),
+  }).then((response) => response.json());
+}
+
+function Login() {
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [response_message, setresponse_message] = useState("");
+  const [passwordError, setpasswordError] = useState("");
+  const [emailError, setemailError] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    let formvalid = handleValidation();
+    if (formvalid === true) {
+      const response = await loginUser({
+        email,
+        password,
+      });
+
+      if (response["token"]) {
+        window.open("/locations", "_self");
+        const token = response["token"];
```

Comment:
```suggestion
        const token = response.token;
```
use dot notation in JS